mono/mini/Makefile.am.in

```diff
@@ -826,7 +826,11 @@ version.h: Makefile
 	if test -d $(top_srcdir)/.git; then \
 		(cd $(top_srcdir); \
 			LANG=C; export LANG; \
-			branch=`git branch | grep '^\*' | sed 's/(detached from .*/explicit/' | cut -d ' ' -f 2`; \
+			if test -z "$$ghprbPullId"; then \
+				branch=`git branch | grep '^\*' | sed 's/(detached from .*/explicit/' | cut -d ' ' -f 2`; \
+			else \
+				branch="$$ghprbPullId"; \
```

Comment:
Pick something, and I'll put it in. It's not a difficult change. I'd prefer no spaces though, as I'm using this value in a debian package name/version string